crates/config/src/errors.rs

```diff
@@ -3,6 +3,12 @@
 pub enum ConfigFileError {
     #[error(transparent)]
     IO(#[from] std::io::Error),
+    #[error("no such file or directory: {0}")]
+    NotFound(String),
+    #[error("no such file or directory: {0}")]
+    IncludedFileNotFound(String),
+    #[error("plugin .wasm file not found: {0}")]
```

Comment:
Probably drop the `.wasm` here. Technically I don't think we prevent files from using a different extension and we might already even support loading `.wat` files instead because the underlying `wasmtime` dependency does. On the off-chance someone takes advantage of that for some reason, the error message could be confusing.